cli/dcoscli/node/main.py

```diff
@@ -520,6 +525,34 @@ def _log(follow, lines, leader, slave, component, filters):
     return 0
 
 
+def _metrics(mesos_id, fields, json_=False):
+    """ Get metrics from the specified agent.
+
+    :param mesos_id: mesos node id
+    :type mesos_id: str
+    :param fields: a list of fields
+    :type fields: [str]
+    :param json_: print raw JSON
+    :type json_: bool
+    """
+
+    endpoint = '/system/v1/agent/{}/metrics/v0/node'.format(mesos_id)
+
+    dcos_url = config.get_config_val('core.dcos_url').rstrip('/')
+    if not dcos_url:
+        raise config.missing_config_exception(['core.dcos_url'])
+
+    url = dcos_url + endpoint
+
+    if fields:
+        return metrics.print_node_metrics_fields(url, fields)
+
+    if json_:
+        return metrics.print_node_metrics_json(url)
+
+    return metrics.print_node_metrics_summary(url)
```

Comment:
can we share code between these methods? They are fairly similar, and all call `_fetch_node_metrics(url)`